.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -46,6 +46,14 @@ In this section, note any new environment variables or feature flags introduced.
 
 -->
 
+## If any indexes were added, were they added to `INDEXES.md`?
+
+<!--
+
+In this section, check the `INDEXES.md` at the root of the repo and make sure you have added and commited any index changes necessary for this PR to be deployed. If you added any, indicated `Yes`in this section and list the index entries you added here.
+
+-->
```

Comment:
@tessalt, I have updated the PR template here with this, right under the env var and feature flags part. That way all the deployment "gotchas" are in one place together on the PR descriptions.